store/models.py

```diff
@@ -0,0 +1,122 @@
+    gender=models.IntegerField(null=True, choices=GENDER)
+    mobile=models.IntegerField(default=0, null=True)
+    address = models.CharField(max_length=255)
+    profile_pic= models.ImageField(upload_to='profile_pic/', null=True, blank=True)
+    city = models.CharField(max_length=100, null=True)
+    postal_code = models.CharField(max_length=20, null=True)
+    country = models.CharField(max_length=50, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    
+    USERNAME_FIELD= 'email'
+    REQUIRED_FIELDS = []
+
+    def __str__(self):
+        return self.email
+    
+    objects=CustomUserManager()
+
+
+class Product(models.Model): 
+    name = models.CharField(max_length=60) 
+    price = models.DecimalField(default=0, decimal_places=2, max_digits=10) 
+    description = models.CharField( max_length=250, default='', blank=True, null=True) 
+    stock=models.PositiveIntegerField(null=True)
+    image = models.ImageField(upload_to='product_img/', blank=True, null=True)
+    quantity = models.IntegerField(default=1) 
+    created_by = models.ForeignKey(User,on_delete=models.CASCADE, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    
+    class Meta:
+        db_table = 'Product_table'
+
+class Wishlist(models.Model):
+    customer = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
+    products = models.ForeignKey(Product, related_name='wishlists', on_delete=models.CASCADE, null=True, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    class Meta:
+        db_table = 'Wishlist_table'
+
+class Order(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.PROTECT, null=True, blank=True) 
+    customer = models.ForeignKey(User, on_delete=models.PROTECT,null=True, blank=True) 
+    required_items = models.BooleanField(default=False)
+    quantity = models.PositiveIntegerField(default=1)
+    total_amount = models.DecimalField(max_digits=10, decimal_places=2, null=True)
+    order_status = models.CharField(max_length=20, choices=ORDER_STATUS, default=PROCESSING)
+    class Meta:
+        db_table = 'Order_table'
+
+class Payment(models.Model):
+    customer = models.ForeignKey(User, 
+                                 on_delete=models.PROTECT,null=True, blank=True) 
+    product = models.ForeignKey(Product, 
+                                on_delete=models.PROTECT, null=True, blank=True) 
+    order = models.ForeignKey(Order, on_delete=models.CASCADE, related_name='payment')
+    transaction_id = models.CharField(max_length=100)
+    payment_method = models.CharField(max_length=20, choices=PAYMENT_OPTION, default=CASH_ON_DELIVERY)
+    delivery_charges = models.PositiveBigIntegerField()
+    payment_status = models.CharField(max_length=20, choices=PAYMENT_STATUS, default=PENDING)
+    created_at = models.DateTimeField(auto_now_add=True)
+    class Meta:
+        db_table = 'Payment_table'
+
+class OrderHistory(models.Model):
```

Comment:
What is the need of this table

Reply:
Table removed. We'll manage it through order(Order Status)